worker/worker-manager/score-worker-manager-impl/src/test/java/org/openscore/worker/management/services/WorkerExecutionMonitorServiceImplTest.java

```diff
@@ -0,0 +1,57 @@
+package org.openscore.worker.management.services;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Matchers;
+import org.openscore.events.EventBus;
+import org.openscore.events.EventConstants;
+import org.openscore.events.ScoreEvent;
+import org.openscore.worker.management.monitor.ScheduledWorkerMonitor;
+import org.openscore.worker.management.monitor.WorkerMonitors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.io.Serializable;
+import java.util.HashMap;
+
+import static org.mockito.Mockito.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = WorkerExecutionMonitorServiceImplTest.MyTestConfig.class)
+public class WorkerExecutionMonitorServiceImplTest {
+    @InjectMocks

```

Comment:
I don't think that you need the inject mocks annotation if you're using spring runner and mocking in the configuration class